controller failed to start because of bad configuration

[vagrant@cfc-master Deploy_OpenWhisk_with_Helm]$ kubectl logs controller-0 -n openwhisk
[2017-10-09T07:09:51.013Z] [INFO] Slf4jLogger started
[2017-10-09T07:09:51.453Z] [INFO] [??] [Config] environment set value for db.whisk.actions
[2017-10-09T07:09:51.454Z] [INFO] [??] [Config] environment set value for limits.actions.sequence.maxLength
[2017-10-09T07:09:51.454Z] [INFO] [??] [Config] environment set value for limits.triggers.fires.perMinute
[2017-10-09T07:09:51.455Z] [INFO] [??] [Config] environment set value for db.protocol
[2017-10-09T07:09:51.455Z] [INFO] [??] [Config] environment set value for loadbalancer.invokerBusyThreshold
[2017-10-09T07:09:51.455Z] [INFO] [??] [Config] environment set value for controller.instances
[2017-10-09T07:09:51.455Z] [INFO] [??] [Config] environment set value for limits.actions.invokes.concurrent
[2017-10-09T07:09:51.455Z] [INFO] [??] [Config] environment set value for whisk.version.date
[2017-10-09T07:09:51.456Z] [INFO] [??] [Config] environment set value for db.port
[2017-10-09T07:09:51.456Z] [INFO] [??] [Config] environment set value for whisk.version.buildno
[2017-10-09T07:09:51.457Z] [INFO] [??] [Config] environment set value for db.username
[2017-10-09T07:09:51.457Z] [INFO] [??] [Config] environment set value for db.whisk.activations
[2017-10-09T07:09:51.457Z] [INFO] [??] [Config] environment set value for limits.actions.invokes.perMinute
[2017-10-09T07:09:51.458Z] [INFO] [??] [Config] environment set value for db.whisk.auths
[2017-10-09T07:09:51.458Z] [INFO] [??] [Config] environment set value for runtimes.manifest
[2017-10-09T07:09:51.458Z] [INFO] [??] [Config] environment set value for kafka.host.port
[2017-10-09T07:09:51.458Z] [INFO] [??] [Config] environment set value for limits.actions.invokes.concurrentInSystem
[2017-10-09T07:09:51.459Z] [INFO] [??] [Config] environment set value for db.host
[2017-10-09T07:09:51.459Z] [INFO] [??] [Config] environment set value for port
[2017-10-09T07:09:51.459Z] [INFO] [??] [Config] environment set value for db.provider
[2017-10-09T07:09:51.459Z] [INFO] [??] [Config] environment set value for db.password
[2017-10-09T07:09:51.460Z] [INFO] [??] [Config] environment set value for kafka.host
[2017-10-09T07:09:51.470Z] [ERROR] [??] [Config] required property controller.localBookkeeping still not set
[2017-10-09T07:09:51.470Z] [ERROR] [??] [Controller] Bad configuration, cannot start.

--- a/templates/controller_statefulset.yml
+++ b/templates/controller_statefulset.yml
@@ -126,4 +126,9 @@ spec:
 
         - name: "CONTROLLER_INSTANCES"
           value: {{ .Values.controllerInstances | default 1 | quote }} 
-
+        - name: "CONTROLLER_LOCALBOOKKEEPING"
+          value: {{ .Values.controllerLocalbookkeeping | default false | quote }}
+        - name: "CONTROLLER_BLACKBOXFRACTION"
+          value: {{ .Values.controllerBlackboxFraction | default 0.10 | quote }}
+        - name: "AKKA_CLUSTER_SEED_NODES"
+          value: {{ .Values.controllerSeedNodes | default "[]" | quote }}

--- a/values.yaml
+++ b/values.yaml
@@ -51,6 +51,9 @@ controllerLimitsTriggersFiresPerminute: 100
 controllerLimitsActionsSequenceMaxlength: 1000
 controllerLimitsActionsInvokesConcurrent: 100
 controllerLimitsActionsInvokesConcurrentinsystem: 100
+controllerLocalbookkeeping: true
+controllerBlackboxFraction: 0.10
+controllerSeedNodes: "[]"
 
 # Invoker configurations
 invokerStatefulsetName: "invoker"

